cmd/nimbletune/nimbletune.go

```diff
@@ -343,6 +344,7 @@ const (
 	xmlDescription        = "XML output of recommendations. (Optional)"
 	verboseDescription    = "Verbose output. (Optional)"
 	versionDescription    = "Display version of the tool. (Optional)"
+	globalDescription     = "Setting iSCSI params per volume instead of at host level in iscsid.conf"
```

Comment:
Please make this generic "If true, settings will be configured globally at host level wherever applicable(eg iscsid.conf), default: false"